freqtrade/freqtradebot.py

```diff
@@ -612,14 +631,17 @@ def execute_entry(self, pair: str, stake_amount: float, price: Optional[float] =
             trade.open_order_id = order_id
 
         trade.orders.append(order_obj)
-        trade.recalc_trade_from_orders()
+        if pos_adjust:
+            trade.recalc_trade_from_orders()
+        else:
+            trade.recalc_open_trade_value()
```

Comment:
This one leaves me with a questionmark for myself (i think it's wrong / pointless).

i think `trade.recalc_trade_from_orders()` should also work for the initial entry - so why do we need to split this?
More specifically - once the order fills, we call `update_trade()` - which for buy orders - will call  `trade.recalc_trade_from_orders()` anyway.

Reply:
Yes, it is not needed here. removed